apps/wallet-mobile/src/Catalyst/ConfirmVotingTx.tsx

```diff
@@ -7,11 +7,11 @@ import {SafeAreaView} from 'react-native-safe-area-context'
 import {KeyboardAvoidingView, ProgressStep, Spacer, TextInput} from '../components'
 import {ConfirmTx} from '../components/ConfirmTx'
 import {debugWalletInfo, features} from '../features'
+import {useSelectedWallet} from '../features/Wallet/common/Context'
```

Comment:
WalletSelection feature is part of WalletManager, it shouldn't be part of the wallet creation/restore.

Reply:
`features/WalletSetup/`

Reply:
done